pkg/plugins/platform/local.go

```diff
@@ -281,9 +279,34 @@ func (l *LocalApp) DockerComposeYAMLPath() string {
 	return l.AppConfig.DockerComposeYAMLPath()
 }
 
+// ComposeFiles returns a list of compose files for a project.
+func (l *LocalApp) ComposeFiles() []string {
+	files, err := util.FindFiles(path.Join(l.AppRoot(), ".ddev"), "docker-compose")
+	if err != nil {
+		log.Fatalf("Failed to load compose files: %v", err)
+	}
+
+	for i, file := range files {
+		// ensure main docker-compose is first
```

Comment:
I'm not quite sure what you're looking for here. Our only concern with ordering is to ensure our main compose file comes first so it may be fully overridden or extended, and to ensure `docker-compose.override.yaml` is last so it can fully override or extend everything before it. I'm not sure how we could provide any more meaningful ordering when we don't really know what files will come in between. It could be that a future plugin needs some more explicit ordering, but this would be handled in that plugin's own ComposeFiles method.